tools/go.mod

```diff
@@ -3,16 +3,24 @@ module github.com/buildpacks/lifecycle/tools
 go 1.15
 
 require (
+	github.com/Azure/azure-sdk-for-go v42.3.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.10.2 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
```

Comment:
I saw these changes too when updating the go.mod and I'm not sure what to make of them. Eventually I was able to get the go.mod to have only the diff that I was expecting (imgutil + ggcr) but I don't understand what happened. I feel like, maybe we should investigate, or maybe it doesn't matter - could someone who understands go.mod a bit better opine here? cc @ekcasey

Reply:
I was able to clean `go.mod` as well.